status.json

```diff
@@ -5081,5 +5081,21 @@
       "priority": "Low"
     },
     "uservoiceid": 19594159
+  },
+  {
+    "name": "Media Queries: prefers-reduced-motion",
+    "category": "Accessibility",
+    "link": "https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion",
+    "summary": "Used to detect if the user has requested the system minimize the amount of animation or motion it uses.",
+    "standardStatus": "Editor’s draft",
```

Comment:
[`’` => `'`](https://travis-ci.org/MicrosoftEdge/Status/builds/361769174#L476)

(I'll fix it before merging)

Reply:
Why change? That's the correct punctuation mark.

Reply:
@melanierichards Feel free to open a PR that changes the schema and all the other values.

---

As far as I remember, if we change the schema and all the other values, it shoundn't braking anything else. (Cc @molant)

Reply:
Ah, gotcha.